plutus-starter.cabal

```
@@ -21,7 +21,26 @@ maintainer:         Your email
 -- category:
 -- extra-source-files: CHANGELOG.md
 
+flag defer-plugin-errors
+    description:
+        Defer errors from the plugin, useful for things like Haddock that can't handle it.
+    default: False
+    manual: True
+
+common lang
+    default-language:   Haskell2010
+    ghc-options:
+      -Wall -Wnoncanonical-monad-instances
+      -Wincomplete-uni-patterns -Wincomplete-record-updates
+      -Wredundant-constraints -Widentities -rtsopts
+      -- See Plutus Tx readme
+      -fobject-code -fno-ignore-interface-pragmas -fno-omit-interface-pragmas
+    -- See Plutus Tx readme
+    if flag(defer-plugin-errors)
+        ghc-options: -fplugin-opt PlutusTx.Plugin:defer-errors
```

I don't see anything about it in the `plutus-tx` README. Is it said implicitly?

---

This was probably goes all the way back to when plutus-starter used to be part of plutus? and that probably came from here intially https://github.com/input-output-hk/plutus/blob/1afef4159d356f3493b7c38ec7c5bb999e75f3b3/plutus-use-cases/plutus-use-cases.cabal#L64-L68

---

Still true for the first comment, but the second one is not needed.

```suggestion
    if flag(defer-plugin-errors)
        ghc-options: -fplugin-opt PlutusTx.Plugin:defer-errors
```

---

oh lol yeah some bad shuffling around